From 30184b429f80393b27931ca8a4233f860bf8db5d Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Mon, 18 Nov 2024 15:04:26 +0100
Subject: [PATCH 1375/1695] dts: stm32: rename scmi regulators by regulator@0
 for stm32mp21x

Use a scmi regulator node name coherent with scmi binding:
protocol@17{
	regulators {
		regulator@0 {
		};
		[...]
	};
};

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 1b99c199c93075a8a8b7b40fff92bfdda8c1cf29 dts: stm32: rename scmi regulators by regulator@0 for stm32mp21x boards
269c3d9e2f3d83ae7c2ba76ecca3d3453d637f51 dts: stm32: rename scmi regulators by regulator@0 for stm32mp21x]
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: I916ba4380cdf7bc49cf3219ce2e95d2f336922bc
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/419308
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 core/arch/arm/dts/stm32mp211.dtsi | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp211.dtsi b/core/arch/arm/dts/stm32mp211.dtsi
index 36a96aaec..522ce1508 100644
--- a/core/arch/arm/dts/stm32mp211.dtsi
+++ b/core/arch/arm/dts/stm32mp211.dtsi
@@ -1024,26 +1024,26 @@
 			scmi_regulator: protocol@17 {
 				reg = <0x17>;
 
-				scmi_regu: regulators {
+				regulators {
 					#address-cells = <1>;
 					#size-cells = <0>;
 
-					voltd-vddio1 {
+					regulator@0 {
 						reg = <VOLTD_SCMI_VDDIO1>;
 						voltd-supply = <&vddio1>;
 					};
 
-					voltd-vddio2 {
+					regulator@1 {
 						reg = <VOLTD_SCMI_VDDIO2>;
 						voltd-supply = <&vddio2>;
 					};
 
-					voltd-vddio3 {
+					regulator@2 {
 						reg = <VOLTD_SCMI_VDDIO3>;
 						voltd-supply = <&vddio3>;
 					};
 
-					voltd-vdda18adc {
+					regulator@3 {
 						reg = <VOLTD_SCMI_ADC>;
 						voltd-supply = <&vdda18adc>;
 					};
-- 
2.47.2

